web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingRoutes.kt

```diff
@@ -112,6 +112,12 @@ internal fun Route.behandlingRoutes(
         }
     }
 
+    get("$behandlingPath/{behandlingId}/utledetSatsInfo") {
+        call.withBehandling(repo) { behandling ->
+            call.svar(OK.utledetSatsJsonBody(behandling))
```

Comment:
Kunne vi heller kanskje ha gjort noe ala
```suggestion
call.svar(OK.jsonBody(UtledetSatsInfoJson(behandling.getUtledetSatsBeløp())))
```
?

Jeg er ikke så superfan av å gjemme alt mulig rundt omkring i extension-metoder.

Reply:
jsonBody er allerede en extension function for å sende behandling JSON, og forslaget over kommer til å gi feil typing. Siden vi i dette tilfellet bare sender satsbeløpet så langt, så er det ikke et problem å egentlig endre hele responsen. 
tanken var at vi har da muligheten til å legge til flere ting etterhvert.

Reply:
`Resultat.json(OK, serialize(behandling.toUtledetSatsInfoJson())`